cmake/kokkos_tribits.cmake

```diff
@@ -160,6 +160,11 @@ FUNCTION(KOKKOS_ADD_EXECUTABLE_AND_TEST ROOT_NAME)
             )
         ENDIF()
     ENDIF()
+    # We noticed problems with -fvisibility-hidden for inline static variables
```

Comment:
We could do: [CXX_VISIBILITY_PRESET](https://cmake.org/cmake/help/latest/prop_tgt/LANG_VISIBILITY_PRESET.html) it's slighlty more idomatic

Reply:
Also, the comment here is a bit weird, since it looks like we are enabling hidden visibility by default (which is a good idea, since it's keeps visibility the same as MSVC)